images/scripts/rhel4edge.bootstrap

```diff
@@ -12,8 +12,13 @@ cat > edge-request.json <<EOF
   "imageType": "rhel-edge-installer",
   "packages": [
     { "name": "cockpit-system" },
+    { "name": "insights-client" },
+    { "name": "ncurses" },
```

Comment:
This is rather non-obvious, and needs a "why?" comment.

Reply:
It provides `/usr/bin/clear` which we call in terminal tests: https://github.com/cockpit-project/cockpit/blob/main/test/verify/check-system-terminal#L72

Reply:
I see; do you mind adding that as a comment?

Also, we don't need ncurses for that, `printf "\033c"` is perfectly sufficient for that. We could also just change that in the test.

Reply:
Added comment.